### DISCUSSION TITLE
Pull Random Bible Quote in Daily Notes

### DISCUSSION THREAD

**Discussion by SuperSonicSites** [SilentVoid13/Templater#263] [Help]
Hello everyone! I just discovered Obsidian and Templater and I'm so amazed!

So, I discovered this interesting project:
https://forum.obsidian.md/t/bible-study-in-obsidian-kit-including-the-bible-in-markdown/12503

In that starter kit there is every books of the bible with every verses separated by a H6.

How would I go about pulling a random quote from that folder? 

For example, if I want to have a quote embedded, I simply type ![[PS-01#v1]]. But it would be great if at the beginning of each day, a random one would be pulled into my daily note.

Can templater do that out of the box?

**Comment by zsviczian** [comment]
check this out: https://www.youtube.com/watch?v=HijK74E3MFM
and the script is here: https://github.com/zsviczian/obsidian-excalidraw-plugin/discussions/144
